Makefile

```
@@ -1,6 +1,31 @@
 # Go parameters
 GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
 GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+BINARY_PATH=build
+BINARY_NAME=$(BINARY_PATH)/smsgateway
+BINARY_UNIX=$(BINARY_NAME)_unix
 
+all: test build
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)